src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Button } from '../../components/Button';
```

Comment:
let's setup aliased imports:
```suggestion
import { Button } from '@/components/Button';
```

https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json
https://koenwoortman.com/vitejs-module-aliases/

Reply:
please create separate PR for this

Reply:
Now i make it resolved, i will vreate new pr